### ISSUE TITLE
Error trying to depend on alacritty_terminal

### ISSUE THREAD

**Issue by RedHatter** [alacritty/alacritty#3411]
I wanted to do some experimentation with alacritty_terminal but I get build errors when I attempt to depend on the library.

```
error: multiple packages link to native library `freetype`, but a native library can be linked only once

package `freetype-sys v0.9.0`
    ... which is depended on by `freetype-rs v0.23.0`
    ... which is depended on by `font v0.1.0 (/home/timothy/Projects/alacritty/font)`
    ... which is depended on by `alacritty_terminal v0.4.2-dev (/home/timothy/Projects/alacritty/alacritty_terminal)`
    ... which is depended on by `alactitty-gtk v0.1.0 (/home/timothy/Projects/alactitty-gtk)`
links to native library `freetype`

package `servo-freetype-sys v4.0.3`
    ... which is depended on by `servo-fontconfig-sys v4.0.9`
    ... which is depended on by `servo-fontconfig v0.4.0`
    ... which is depended on by `font v0.1.0 (/home/timothy/Projects/alacritty/font)`
    ... which is depended on by `alacritty_terminal v0.4.2-dev (/home/timothy/Projects/alacritty/alacritty_terminal)`
    ... which is depended on by `alactitty-gtk v0.1.0 (/home/timothy/Projects/alactitty-gtk)`
also links to native library `freetype`
```

Cargo.toml
```
[package]
name = "alactitty-gtk"
version = "0.1.0"
authors = ["Timothy Johnson <timothy@idioticdev.com>"]
edition = "2018"

[dependencies]
alacritty_terminal = { path = "../alacritty/alacritty_terminal/" }

[dependencies.gtk]
version = "0.8.0"
features = ["v3_16"]

[dependencies.gio]
version = ""
features = ["v2_44"]
```

**Comment by kchibisov** [comment]
IIRC you need to do something like this https://github.com/alacritty/alacritty/blob/f7561850b43cbb1e7a8ccd9e652ef7d338a6cf2b/Cargo.toml#L14 due to dependency on a  font crate. However depending on our font crate in alacritty_terminal doesn't make much sense, so there's an issue about removing such dependency #3393 . So closing as a duplicate of it.
